python/lsst/daf/butler/registries/sqlPreFlight.py

```diff
@@ -489,6 +530,10 @@ def _convertResultRows(self, rowIter, dimensions, linkColumnIndices, regionColum
             # filtered if any of two regions are disjoint.
             disjoint = False
             regions = {holder: Region.decode(row[col]) for holder, col in regionColumnIndices.items()}
+            if "skypix" in linkColumnIndices:
```

Comment:
I know this is something we worked on together but you may want to add a comment talking about why this is needed and or mention a ticket that might fix this.

Reply:
Will add a comment.  While it might be nice to try to unify the special-casing of SkyPix, some special-casing is always going to have to be there.